src/confd/src/ietf-interfaces.c

```diff
@@ -495,7 +496,7 @@ static int netdag_gen_iface_del(struct dagger *net, struct lyd_node *dif,
 
 	DEBUG_IFACE(dif, "");
 
-	mcast_querier(ifname, 0, 0, 0);
+	/* WKZ TODO mcast_querier(ifname, 0, 0, 0); */
```

Comment:
This is the only standout issue I can find.

Reply:
Thanks! :sweat_smile: 

Yes, this is now taken care of by the `mcd` config generation.